apis/nucleus/src/object/initiate-sheet.js

```diff
@@ -1,22 +1,23 @@
 /* eslint no-underscore-dangle:0 */
 import sheetAPI from '../sheet';
+import createNavigationApi from './navigation/navigation';
 
-export default async function initSheet(model, optional, halo, initialError, onDestroy = async () => {}) {
+export default async function initSheet(model, optional, halo, store, initialError, onDestroy = async () => {}) {
   const api = sheetAPI({
     model,
     halo,
     initialError,
     onDestroy,
   });
```

Comment:
I think it looks better to pass `navigation` in here, and not into the `mount()` call. We don't want a new navigation API instance for each mount call?

Reply:
Good idea! Fixed! Thanks!

Reply:
To support for single nav-menu object case, the navigation creation is moved up to apis/nucleus/src/object/get-generic-object.js and from that pass navigation to either a sheet or a single nav-menu object.